### ISSUE TITLE
404 error for the readme "the template compose file" link

### ISSUE THREAD

**Issue by Starlyvil** [itsthejb/docker-ispconfig#375]
The link to the template which is supposed to be copied to the standard filename of **docker-compose.yml** is broken on the readme page, making it hard to following the installation guide.

![image](https://github.com/itsthejb/docker-ispconfig/assets/31422222/47c521be-43da-46ec-bad4-2caf8d91a889)

Please kindly help fix this

**Comment by itsthejb** [comment]
Hi there,

Thanks for your input. I've just updated the link, and slightly updated the instructions. Please note that this repo isn't exactly supported, so I can't make any promises about ease of use! That said, I'll be happy if it's useful and would welcome any changes to the setup instructions that might make this more approachable - however, please don't expect it to be easy to use out-of-the-box!

Thanks

**Comment by Starlyvil** [comment]
Thanks @itsthejb I really appreciate it
